src/components/organisms/Portfolio/index.tsx

```diff
@@ -38,41 +40,32 @@ const Portfolio = () => {
   };
 
   const getSlideData = (id: number) => {
-    switch (id) {
-      case 2:
-        return (
-          <div id="screenshot">
-            <EducationDetail />
-          </div>
-        );
-
-      case 4:
-        return (
-          <div id="screenshot">
-            <Certifications />
-          </div>
-        );
+    if (id === 2) {
+      return <EducationDetail />;
+    }
+    else {
+      return <Certifications />;
```

Comment:
It should return if` id !== 2 `or `id === 4`? Check the condition in previous commits

Reply:
actually we have only two organisms to render one with id = 2 and other with id = 4. So if id == 2 it will return first organisms or else the second one

Reply:
![image](https://user-images.githubusercontent.com/55446681/191183741-ea047985-65ef-4d9d-a871-009792259bd9.png)

If id == 2 Education Details will be render otherwise Certifications will be rendered. All other components are out of scope for me

Reply:
Okay